jekyll/assets/css/main.scss

```diff
@@ -148,13 +161,14 @@ body.loggedin {
 
 // Article
 div.article-container{
-	float: left;
-	@media screen and (max-width: $screen-xs-max) {
-		padding: 0 16px 0 16px;
-		width: 100%;
-	}
-	padding: 0 48px;
-	width: calc(100% - 260px);
+  flex: 1;
+  @media screen and (max-width: $screen-xs-max) {
+      padding: 0 16px 0 16px;
+      width: 100%;
+  }
+  padding: 0 48px;
+	width: calc(100% - #{$rightbar-width});
+  max-width: 60em; // increase readability on large screens.
```

Comment:
More indentation weirdness here, inconsistency between this line and the one before it.

Reply:
Fixed